unit-testing/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+      "target": "es6",                          // Specify ECMAScript target version
+      "module": "commonjs",                     // Specify module code generation
+      "noImplicitAny": false,  
+      "strict": false,                           // Enable all strict type-checking options
+      "esModuleInterop": true,                  // Enables emit interoperability between CommonJS and ES Modules
+      "skipLibCheck": true,                     // Skip type checking of declaration files
+      "forceConsistentCasingInFileNames": true, // Disallow inconsistently-cased references to the same file
+      "types": ["jest", "node"],                // Specify type package
+      "outDir": "./dist"
+    },
+    "include": ["src"],                         // Specify which files to include
+    "exclude": ["node_modules"], // Specify which files to excludeS

```

```suggestion
    "exclude": ["node_modules"], // Specify which files to exclude
```

Although I think you can remove both `include` and `exclude` here if you set `rootDir` to `src` - I don't recall 100% so feel free to ignore me if I'm wrong. You may not need `skipLibCheck` either and can probably turn `strict` to `true`. Once you do that, `noImplicitAny` defaults to `true` which I think is better anyway so we can remove `noImplicitAny` on line 5 as well

---

Made these changes. Required some touch up in .ts files.